test/jdk/java/nio/file/Path/PathOps.java

```diff
@@ -2042,6 +2042,12 @@ static void doUnixTests() {
     static void npes() {
         header("NullPointerException");
 
+        try {
+            Path.of(null, "foo");
+            throw new RuntimeException("NullPointerException not thrown");
+        } catch (NullPointerException npe) {
+        }
+
```

Comment:
Probably should test Path.of("foo", null) too, just so we have tests for null at the first and second parameter in the same test.